examples/calling-apis/chatbot/app/(langgraph)/api/langgraph/[..._path]/route.ts

```diff
@@ -15,15 +15,18 @@ async function getAccessToken() {
 
 export const { GET, POST, PUT, PATCH, DELETE, OPTIONS, runtime } =
   initApiPassthrough({
+    disableWarningLog: true,
     apiUrl: process.env.LANGGRAPH_API_URL,
     apiKey: process.env.LANGSMITH_API_KEY,
     runtime: "edge",
     baseRoute: "langgraph/",
     headers: async (req: NextRequest) => {
-      const headers: Record<string, string> = {};
-      req.headers.forEach((value, key) => {
-        headers[key] = value;
-      });
+      const headers: Record<string, string> = {
+        ...Object.fromEntries(req.headers.entries()),
+      };
+
+      // Remove Next.js session cookie header
+      delete headers.Cookie;
```

Comment:
would it be clearer/safer to only copy the headers that are important to copy?

By being explicit in this example, we can help developers understand the purpose of this code (which header(s) is/are important to copy for this example to work well). wdyt?